contrib/node/install-sdn.sh

```diff
@@ -1,36 +1,32 @@
 #!/bin/bash
 
 os::provision::install-sdn() {
-  local default_target="/usr"
-
   local deployed_root=$1
-  local target=${2:-${default_target}}
+  local target=${2:-}
+  local target_usrdir="${target}/usr"
+  local target_bindir="${target_usrdir}/bin"
+  local target_etcdir="${target}/etc"
+  local target_cnidir="${target}/opt/cni/bin"
 
-  if [ ! -d ${target} ]; then
-    mkdir -p ${target}
-  fi
+  mkdir -p ${target_usrdir}
+  mkdir -p ${target_bindir}
+  mkdir -p ${target_etcdir}
+  mkdir -p ${target_cnidir}
 
   local osdn_plugin_path="${deployed_root}/pkg/sdn/plugin"
-  mkdir -p "${target}/bin/"
   pushd "${osdn_plugin_path}" > /dev/null
-    install bin/openshift-sdn-ovs "${target}/bin/"
-    install bin/openshift-sdn-docker-setup.sh "${target}/bin/"
+    install sdn-cni-plugin/openshift-sdn-ovs "${target_bindir}"
   popd > /dev/null
 
-  # osdn plugin setup writes docker network options to
-  # /run/openshift-sdn/docker-network, make this file to be exported
-  # as part of docker service start.
-  local system_docker_path="${target}/lib/systemd/system/docker.service.d/"
-  mkdir -p "${system_docker_path}"
-  cat <<EOF > "${system_docker_path}/docker-sdn-ovs.conf"
-[Service]
-EnvironmentFile=-/run/openshift-sdn/docker-network
-EOF
```

Comment:
this (and a few other things like deleting `contrib/systemd/docker-sdn-ovs.conf`) should be in the previous commit